From: Borislav Petkov <borislav.petkov@amd.com>
Subject: EDAC, MCE: Add a BIT_64() macro
References: fate#311581
Git-commit: cf1d2200dbc214c26a116c4d0c75b7cf27bb19b6
Patch-mainline: v2.6.37-rc1

Signed-off-by: Thomas Renninger <trenn@suse.de>

Add a macro for 64-bit vectors to use when accessing MSR contents.

Signed-off-by: Borislav Petkov <borislav.petkov@amd.com>
---
 drivers/edac/mce_amd.h |    2 ++
 1 files changed, 2 insertions(+), 0 deletions(-)

diff --git a/drivers/edac/mce_amd.h b/drivers/edac/mce_amd.h
index 0d0637d..35f6e0e 100644
--- a/drivers/edac/mce_amd.h
+++ b/drivers/edac/mce_amd.h
@@ -5,6 +5,8 @@
 
 #include <asm/mce.h>
 
+#define BIT_64(n)			(U64_C(1) << (n))
+
 #define ERROR_CODE(x)			((x) & 0xffff)
 #define EXT_ERROR_CODE(x)		(((x) >> 16) & 0x1f)
 
-- 
1.7.5.3

